In [1]:
import pandas as pd

In [3]:
df = pd.read_csv("test.csv")
df.head()

PassengerId  Pclass                                          Name     Sex  \
0          892       3                              Kelly, Mr. James    male   
1          893       3              Wilkes, Mrs. James (Ellen Needs)  female   
2          894       2                     Myles, Mr. Thomas Francis    male   
3          895       3                              Wirz, Mr. Albert    male   
4          896       3  Hirvonen, Mrs. Alexander (Helga E Lindqvist)  female   

    Age  SibSp  Parch   Ticket     Fare Cabin Embarked  
0  34.5      0      0   330911   7.8292   NaN        Q  
1  47.0      1      0   363272   7.0000   NaN        S  
2  62.0      0      0   240276   9.6875   NaN        Q  
3  27.0      0      0   315154   8.6625   NaN        S  
4  22.0      1      1  3101298  12.2875   NaN        S

Pasajeros

In [ ]:
df["Pclass"].value_counts()

Pclass
3    218
1    107
2     93
Name: count, dtype: int64

dentro de la clase 3 hay 218 pasajeros, de la clase 2 hay 93 pasajeros y de la primera clase solo hay 107


In [6]:
df.groupby("Sex")["Age"].mean().round(2)


Sex
female    30.27
male      30.27
Name: Age, dtype: float64

ambos tienen un promedio de 30.27 años de edad